test/AWS.Logger.SeriLog.Tests/AWSSeriLogGroupRestrictedToMinimumLevel.json

```diff
@@ -0,0 +1,18 @@
+﻿{
+  "Serilog": {
+    "Using": [ "AWS.Logger.SeriLog" ],
+    "MinimumLevel": "Debug",
```

Comment:
Why would `MinimumLevel` be **Debug** but `restrictedToMiniumLevel` be **Error**?  Is this a valid configuration case? Is the order of operations documented?

Reply:
`restrictedToMiniumLevel` overrides the behavior for a sink. This is a valid configuration. If you refer the test case, we get 6 log lines from CloudWatch (for Error), instead of 10 (which would have included Debug).